src/__tests__/useCountUp.test.js

```diff
@@ -179,13 +200,14 @@ describe('useCountUp', () => {
 
     act(() => {
       pauseResumeFn();
-    })
+    });
 
-    await checkContent(container, '10')
+    await checkContent(container, '10');
   });
 
   it('re-renders for new props', async () => {
-    const Hook = ({ suffix = '', prefix = '', separator= '', end = 10 }) => {
+    // eslint-disable-next-line react/prop-types
```

Comment:
i think that we can just completely disable this rule

Reply:
It is actually disabled and I added this line before ultimately disabling it. It can be removed.